commands/registry.go

```diff
@@ -144,6 +144,21 @@ func Repository() *Command {
 		Writer, aliasOpt("ls"), displayerType(&displayers.Repository{}),
```

Comment:
```suggestion
		Writer, aliasOpt("ls"), displayerType(&displayers.Repository{}),
		hiddenCmd(),
```

FYI: If we want to push users towards the v2 command, it is possible to hid the current one from menus while leaving it available in order to not break any scripting.